package.json

```diff
@@ -61,14 +61,14 @@
     "@babel/preset-env": "^7.16.4",
     "@babel/preset-react": "^7.16.0",
     "@babel/preset-stage-2": "^7.8.3",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.5",
```

Comment:
There's no official adapter yet, see https://github.com/enzymejs/enzyme/issues/2429

Reply:
That's not great, but oh well 🤷‍♂️